oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConstants.java

```diff
@@ -38,28 +39,41 @@ private OracleConstants() {
   public static final String NAME_DATABASE = "database";
   public static final String TNS_CONNECTION_TYPE = "TNS";
   public static final String TRANSACTION_ISOLATION_LEVEL = "transactionIsolationLevel";
+  public static final String USE_SSL = "useSSL";
 
   /**
-   * Returns the Connection String for the given ConnectionType.
+   * Constructs the Oracle connection string based on the provided connection type, host, port, and database.
+   * If SSL is enabled, the connection protocol will be "tcps" instead of "tcp".
    *
    * @param connectionType TNS/Service/SID
    * @param host Host name of the oracle server
    * @param port Port of the oracle server
    * @param database Database to connect to
-   * @return Connection String based on the given ConnectionType
+   * @param useSSL         Whether SSL/TLS is required(YES/NO)
+   * @return Connection String based on the given parameters and connection type.
    */
   public static String getConnectionString(String connectionType,
```

Comment:
Since we are more focused on the DTS specific changes so we should overload this method, in-order to reduce the impact area i.e. `OracleAction & OraclePostAction`.
ex.
```
public static String getConnectionString(String connectionType,
                                           @Nullable String host,
                                           @Nullable int port,
                                           String database) {
                                           String database) {
     return OracleConstants.getConnectionString(this.connectionType, host, port, database, null);
}
```

Reply:
Overloaded the method, made minor changes to reuse the Connection string format. [3c675f443ee3349528a69b8be4a95edaafe15072]